arm-cdn/2016-10-02/swagger/cdn.json

```diff
@@ -731,14 +731,14 @@
           {
             "name": "endpointName",
             "in": "path",
-            "description": "Name of the endpoint under the profile which is unique globally.",
+            "description": "Name of the endpoint under the profile which is unique globally.", 
             "required": true,
             "type": "string"
           },
           {
             "name": "contentFilePaths",
             "in": "body",
-            "description": "The path to the content to be purged. Path can describe a file or directory using the wildcard. e.g. '/my/directory/*' or '/my/file.exe/'",
+            "description": "The path to the content to be purged. Path can be a full URL, e.g. ‘/pictires/city.png' which removes a single file, or a directory with a wildcard, e.g. ‘/pictures/*' which removes all folders and files in the directory.",
```

Comment:
- please change the single quote in '/pictures/city.png' and '/pictures/*' to a single quote used in programming languages. Otherwise, this will generate wonky stuff in the generated client doc and also on the doc website.
- Incorrect spelling of pictures in ‘/pictires/city.png'